### DISCUSSION TITLE
Compatibility with intel iris xe graphics video card

### DISCUSSION THREAD

**Discussion by cholachbuca** [prasanthrangan/hyprdots#255] [Q&A] [answered]
Hello I have a question, I am a new user of the arch linux system and this configuration of hyprland seemed very complete and I wanted to try it but it mentioned that it was for nvidia cards but it was installed well in my laptop with an intel iris xe card and I wanted to ask if having a different card to the recommended one could damage my video card.

**Comment by T-Crypt** [comment, marked as answer]
@cholachbuca 

Its not going to damage your video card -- since you have Intel IRIS XE that is built into the same chip as your CPU -- So its just going to handle all of the graphics processing for your system. It will not damage it but it won't be recommended for gaming or high video processing programs / applications. 

Im working on getting the temperature of Intel Integrated GPU's currently in my gpuinfo.sh script, and you can monitor the progress there if you would like!

#225

**Reply by cholachbuca** [reply]
Ok, thank you very much, but something else, do you know why I see some pixelated programs or how I can change that?

**Reply by T-Crypt** [reply]
Can you elaborate? 

If its pixelated you may need to adjust your screen resolution to your monitor specs.

Can you run `hyprctl monitors` and paste the output here?

**Reply by cholachbuca** [reply]
ok this is what I get:

Monitor eDP-1 (ID 0):
	1920x1080@60.00100 at 0x0
	description: BOE 0x08D5 (eDP-1)
	make: BOE
	model: 0x08D5
	serial: 
	active workspace: 1 (1)
	special workspace: 0 ()
	reserved: 0 0 0 28
	scale: 1.50
	transform: 0
	focused: yes
	dpmsStatus: 1
	vrr: 0

**Reply by T-Crypt** [reply]
Can you use Super + P to snip the pixelated apps and upload that here as well?

**Reply by cholachbuca** [reply]
well I haven't installed several programs yet but I guess they will look the same as these two which are vscode and microsoft edge, the only program that looks good is firefox but I don't know why.

![230922_14h09m34s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/77275000/9be86fa4-3f90-40ba-9d97-6b2bd0a1203d)

![230922_14h04m32s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/77275000/56927106-6d55-4222-920e-91a366ae6d53)

![230922_14h01m38s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/77275000/2a6e44c6-ba29-4ccf-a343-74fb96aa3160)

**Reply by Senshi111** [reply]
my configuration on my laptop as far as scaling is 

monitor = ,preferred,auto,1.25

# unscale XWayland
xwayland {
  force_zero_scaling = true
}

# toolkit-specific scale
env = GDK_SCALE,2
env = XCURSOR_SIZE,32
#env = GDK_DPI_SCALE=1

problem is xwayland apps not scaling good so you set  not to scale xwayland apps
another way is to set --ozone-platform=wayland on run to xwayland apps
of for example set ozone platform in app to true let say in chome and chrome based browsers


Go to chrome://flags

Search "Preferred Ozone platform"

Set it to "Wayland"

**Comment by T-Crypt** [comment]
This is due to your scaling of 1.5 

I have had the same issue when using any scaling with a decimal place.

Scaling 1 looks good, 2 looks good but 1.25 and 1.5 look blurry. I think this is for most xwayland apps cause even Microsoft edge becomes blurry with 1.25 scaling

you can adjust the scaling of your monitor in the monitor.conf or in the hyprland.conf.

for you it would need to be this in your monitors.conf 


 █▀▄▀█ █▀█ █▄░█ █ ▀█▀ █▀█ █▀█ █▀
 █░▀░█ █▄█ █░▀█ █ ░█░ █▄█ █▀▄ ▄█


# monitor = ,preferred,auto,auto
monitor = eDP-1, 1920x1080@60, 0x0, 1

**Reply by cholachbuca** [reply]
now looks better, thanks

**Reply by T-Crypt** [reply]
Will you just make sure you select that this is answered so it does not show unanswered! 

** Feel free to reach out if you have any more issues or just need help.
